scraper.py

```diff
@@ -76,7 +76,7 @@ def get_page_soup(url:str) -> BeautifulSoup:
     return page_soup
 
 
-def get_urls(category_url:str, category:str, time_delay:bool) -> list:
+def get_urls(category_url:str, category:str, time_delay:bool, articles_per_category: int = None) -> list:
```

Comment:
pls import `Optional` from `typing` and modify this to show that the default is `None`. 

So it becomes `articles_per_category: Optional[int] = None`